### PR TITLE
starting the 2plus2 stuff

### LINKED ISSUE #5253: Design enhancements for 2plus2 pages
Way to call out that a visitor is on a 2plus2 page vs an Area of Study page (url structure "/2plus2/" including updated breadcrumbs, maybe banner with pre-title?)

### PR DESCRIPTION
**Pull request by GaryRidgway**
This adds the banner and pre-title for a 2plus2 major.

# How to test

composer install.
Sync admissions.
`yarn workspace admissions_core sass-compile && ddev drush @admissions.local cr`
create a community college taxonomy term.
create a piece of `Major (2 plus 2)` content, using your Community college and any areas of study that you would like.
Add a featured image to this piece of content.
Verify that the pre-title has a yellow background and black text, and that the title has white text over top of the banner picture.
Verify that your community college is brought in correctly into the title

### CODE REVIEW

**Comment by GaryRidgway** [comment]
This is currently what the pre-title looks like. I think it might be a bit small, but I know that @bspeare is meeting with some people tomorrow for some context, so I shall await their decisions on font type and size!
Screen Shot 2022-05-19 at 11 10 31 AM

**Comment by cory-skeers** [comment]
Do we need to account for the possibility of not having a featured image?
image
I think from discussions the plan is to always have one, at least a common image across all majors.

**Comment by cory-skeers** [comment]
> I think from discussions the plan is to always have one, at least a common image across all majors.

Quick discussion on this, and the plan is to have a common image for every major. Updating to require adding the featured image. Further updates may come later to use a default image if one isn't provided, but for now going with requiring as the lowest-cost solution.

File: docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/field--node--title--centered-highlighted.html.twig [hunk 1]
```diff
@@ -42,9 +42,17 @@
   'text-align-center',
   'headline',
   'headline--uppercase',
-  'headline--highlight',
+  'headline--negative',
```
Comment (bspeare) [inline docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/field--node--title--centered-highlighted.html.twig]: This changes the style on the areas of study pages such as https://admissions.uiowa.ddev.site/academics/accounting.   It looks like we either need a new template or preprocess this class in for 2plus2 majors only.

File: docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/field--node--title--centered-highlighted.html.twig [hunk 2]
```diff
@@ -42,9 +42,17 @@
...
 ]
 %}
+<p>
+{#  {{ devel_breakpoint() }}#}
```
Comment (bspeare) [inline docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/field--node--title--centered-highlighted.html.twig]: Should we remove this `devel_breakpoint`?

**Comment by bspeare** [comment]
I've made a few style updates that I noticed and some changes from our meeting with Scott.   @cory-skeers do you want to test again and approve?

**Review by cory-skeers — APPROVED** [review]
Reviewed and tested the latest updates, and it's looking good!

**Comment by bspeare** [comment]
Thanks! I'm going to merge this in and then deploy https://github.com/uiowa/uiowa/pull/5306 to `dev` to set up the demo content.

**Pull request merged**